tests/test_all_fields.py

```diff
@@ -55,12 +48,34 @@
     'plans': set(),
     'invoice_line_items': {
         'tax_rates',
-        'unique_id',
```

Comment:
The response of the latest API(2020-08-27) version does not return the `unique_id` field.